### ISSUE TITLE
OutlookApi - Can't find events (like Outlook.Items.ItemAdd)

### ISSUE THREAD

**Issue by dp250195** [NetOfficeFw/NetOffice#344]
I can't find a way to attach an event handler to `Outlook.Items.ItemAdd`. Visual Studio IntelliSense doesn't show any event handlers available for `Outlook.Items`. I've gone through all available examples, but I can't find any that illustrate use of event handlers. I tried adding `using NetOffice.OutlookApi.Events`, but this doesn't seem to add anything. Searching the entire Internet did not provide any useful answers.

**Comment by dp250195** [comment]
> Does this solution help? https://stackoverflow.com/questions/41387975/netoffice-outlook-add-in-access-folder-events

Unfortunately, this does not help.

I can see the event being declared here: https://github.com/NetOfficeFw/NetOffice/blob/main/Source/Outlook/Events/ItemsEvents.cs#L22 but I can't find a way to reference it in my code. I'm not making an AddIn though, I'm making a Windows Service app, I added the NuGet packages myself and I did not use this Toolbox being mentioned.
